### ISSUE TITLE
compressed image support

### ISSUE THREAD

**Issue by eadmaster** [meefik/linuxdeploy#608]
I'd like to suggest adding compressed image support using btrfs to save some storage space.

**Comment by meefik** [comment]
Btrfs must be maintained in the core Android but this is usually not the case.

**Comment by eadmaster** [comment]
You're right about that, what about using some fuse-based filesystem then?
There is also f2fs which is well-supported, but it is not clear to me how if it supports transparent compression or not...
